File: src/TechmindsCRM/TechmindsCRM.Core/Models/Cliente.cs
using TechmindsCRM.Commom.Abstract;

namespace TechmindsCRM.Core.Models
{
    public class Cliente : Entidade
    {
        public string Nome { get; set; }

        public string CPFCNPJ { get; set; }

        public string Telefones { get; set; }

        public string Email { get; set; }

        public string CEP { get; set; }

        public string Logradouro { get; set; }

        public string Complemento { get; set; }

        public long Numero { get; set; }

        public string Bairro { get; set; }

        public string Municipio { get; set; }

        public string UF { get; set; }
    }
}

File: src/TechmindsCRM/TechmindsCRM.Web/App/models/Cliente.js
define(["knockout"], function (ko) {
    function ClienteViewModel(data) {
        data = data || {};
        this.id = ko.observable(data.id || 0);
        this.nome = ko.observable(data.nome);
        this.cpfcnpj = ko.observable(data.cpfcnpj).extend({ cpfcnpjExtender: {} });
        this.telefones = ko.observable(data.telefones);
        this.email = ko.observable(data.email);
        this.cep = ko.observable(data.cep);
        this.logradouro = ko.observable(data.logradouro);
        this.complemento = ko.observable(data.complemento);
        this.numero = ko.observable(data.numero);
        this.bairro = ko.observable(data.bairro);
        this.municipio = ko.observable(data.municipio);
        this.uf = ko.observable(data.uf);
    }

    return ClienteViewModel;
});